### ISSUE TITLE
NuGet package of HealthChecks

### ISSUE THREAD

**Issue by CESARDELATORRE** [aspnet/HealthChecks#34]
As soon as we have a first stable preview version, it would be good to have a NuGet package of it so samples or real usage of it rely on the packages instead of on GitHub code.
Even if it is a 0.2 version, I think it is worth to start having a NuGet package, soon.

**Comment by rjperes** [comment]
Any idea as to when this will be pushed live? Thanks!

**Comment by edwardmarbry** [comment]
Bump????? Any time soon?

**Comment by CESARDELATORRE** [comment]
For ASP.NET Core 2.1 timeframe

**Comment by sartan** [comment]
What is the relationship to https://github.com/seven1986/HealthChecks, which is [available via NuGet](https://www.nuget.org/packages/Microsoft.AspNetCore.HealthChecks)?

**Comment by rjperes** [comment]
"Forked from dotnet-architecture/HealthChecks"

**Comment by rjperes** [comment]
The one that is available from NuGet is this one.

**Comment by johnduhart** [comment]
I've reported the NuGet package.

**Comment by sartan** [comment]
Yep, I understand that it's forked from this project, just wasn't clear on the NuGet package since all the info (owner, project site, etc) is for the fork.

So just to be explicit, is the following the official HealthChecks package?
https://www.nuget.org/packages/Microsoft.AspNetCore.HealthChecks

**Comment by johnduhart** [comment]
@sartan There is no official NuGet package, as the library hasn't been released yet.

**Comment by sartan** [comment]
@johnduhart Got it. Thanks for the clarification.

**Comment by CESARDELATORRE** [comment]
Right, I can confirm that. https://www.nuget.org/packages/Microsoft.AspNetCore.HealthChecks is not official but created by "seven".

**Comment by sartan** [comment]
Ah, just came across the PR from seven.

https://github.com/dotnet-architecture/HealthChecks/pull/66

**Comment by filoe** [comment]
If the library is not released yet, why is there no preview version on NuGet?
There are articles like https://docs.microsoft.com/en-us/dotnet/standard/microservices-architecture/implement-resilient-applications/monitor-app-health, but still no NuGet packages. :confused:

**Comment by CESARDELATORRE** [comment]
The official NuGet package for this should appear with the first ASP.NET Core 2.1 Preview.

**Comment by Guymestef** [comment]
FYI, using the nuget package created by @seven1986 with Microsoft.AspNetCore >= 2.0.1, throw an exception at startup: Could not load file or assembly System.ComponentModel.Primitives :/

**Comment by robichaud** [comment]
@CESARDELATORRE Now that ASP.NET Core 2.1 is in Preview, are there any updates on when NuGet packages will be available?

**Comment by Eilon** [comment]
Adding @glennc who I think can comment on the plans for this package. This is not currently planned for ASP.NET Core 2.1, but I believe it is being considered for 2.2.
